Network/Control/Flow.hs

```diff
@@ -81,25 +79,55 @@ txWindowSize TxFlow{..} = txfLimit - txfSent
 
 -- | Flow for receiving.
 --
+-- The goal of 'RxFlow' is to ensure that our network peer does not send us data
+-- faster than we can consume it. We therefore impose a buffer size (also known
+-- as a _window_) of unconsumed bytes that we are willing to receive from the
```

Comment:
In my understanding, *window* is the buffer size which is not filled by peer, not the total size of the buffer.
So, this explanation is a bit confusing.

Reply:
Have rewritten this as 

> The goal of 'RxFlow' is to ensure that our network peer does not send us data faster than we can consume it. We therefore impose a maximum number of unconsumed bytes that we are willing to receive from the peer, which we refer to as the buffer size: